lms/sql_tasks/tasks/fdw/create_from_scratch/00_create.jinja2.sql

```diff
@@ -1,15 +1,31 @@
-DROP SERVER IF EXISTS "{{h_fdw_server_name}}" CASCADE;
```

Comment:
For my money, I think this is part of the `report/create_from_scratch`. I don't think there's a distinction as we need to alter both when we change what we need?